docs/setup/setup.md

```diff
@@ -61,19 +61,22 @@ The cert/key will be generated in the `/etc/kubeedge/ca` and `/etc/kubeedge/cert
 ### Run Cloud
 
 #### Run as a binary
-
++ Before beginning, though, you should be install gcc. Check command:
```

Comment:
Nice catch but we should fix the grammar issue in this sentence :)

Reply:
Please check it again